cmake/Platform/Windows/PackagingPostBuild_windows.cmake

```diff
@@ -102,18 +102,19 @@ file(GLOB _artifacts
     "${_cpack_wix_out_dir}/*.msi" 
     "${_cpack_wix_out_dir}/*.cab"
     "${_cpack_wix_out_dir}/*.exe"
+    "${CPACK_3P_LICENSE_FILE}"
+    "${CPACK_3P_MANIFEST_FILE}"
 )
 file(COPY ${_artifacts}
     DESTINATION ${CPACK_UPLOAD_DIRECTORY}
 )
 message(STATUS "Artifacts copied to ${CPACK_UPLOAD_DIRECTORY}")
 
 if(CPACK_UPLOAD_URL)
-    file(TO_NATIVE_PATH "${_cpack_wix_out_dir}" _cpack_wix_out_dir)
     ly_upload_to_url(
         ${CPACK_UPLOAD_URL}
-        ${_cpack_wix_out_dir}
```

Comment:
I'm running into a cab hash issue with the installer after changing this to `CPACK_UPLOAD_DIRECTORY`. This *shouldn't* run into issues, as we copied the install files in line 109.

Will be testing this out in a another branch in any case

Reply:
Turns out this was due to a recent Cloudfront outage, so unrelated to this change